### ISSUE TITLE
Trouble making breadcrumbs work

### ISSUE THREAD

**Issue by richb-hanover** [comsysto/jekyll-breadcrumb-for-github-pages#3]
I used the breadcrumb snippet from the README.md file in my own project, and it doesn't work. The trouble I see a) it doesn't appear in the page, but this could simply be a CSS problem; b) when I view source, I only see a tag generated for Home, never a link for the current page (see generated HTML below). All pages have `breadcrumb:` in the front matter.

I'm sure that I am doing something wrong, but I notice two differences that might affect things:
1. The snippet in the README.md file is slightly different from the one in gh-pages/_includes/header.html
2. I see the gh-pages/_config.yml file includes some variables that are not present in mine. What variables are required to make breadcrumbs work?

Any other thoughts on getting on the air? Thanks.

**Generated HTML:**

```
<ol class="breadcrumb">
    <li><a href="//">Home</a></li>

<many, many empty lines deleted>  

</ol>
```

**Comment by offboarded-x233534** [comment]
I will check that. I will get back to you once I tested it out.

**Comment by offboarded-x233534** [comment]
Like i wrote in #2 the breadcrumb works fine.
But you need to have a certain file structure with `/pagename/index.html` folders containing index.html

If you stick to the structure of the demo github pages branch you should be fine.
Without seeing the whole project you are using it is hard for me to figure out whats wrong :(

**Comment by richb-hanover** [comment]
I was hoping to have multiple files/pages in a folder, like this:

```
cabins/
   index.html
   rates.html
   pictures.html
```

where each page/file would have its own `breadcrumb: ....` front matter. Would this project handle this?

**Comment by offboarded-x233534** [comment]
I am afraid that won't work. You would have to adapt the code yourself. I want to keep ist simple and provide the breadcrumb as it is now.

The way it works is with this structure (which is the better SEO structure anyway)

```
cabins/index.html
cabins/rates/index.html
cabins/pictures/index.html
```

**Comment by richb-hanover** [comment]
I see... I need that folder structure, so I was able to use a modification of http://stackoverflow.com/questions/9612235/what-are-some-good-ways-to-implement-breadcrumbs-on-a-jekyll-site/37324646#37324646 to get what I needed. Thanks!

**Comment by zgparsons** [comment]
I have a similar issue with the breadcrumbs not generating properly. I believe I am using the correct file structure - the only thing different is that I'm using index.md files in each of my page folders (but they get converted to .html so I don't think this is an issue).

My site is here on github: https://github.com/zgparsons/personal-site

My html is rendering like so:
```
<ol class="breadcrumb">
<li><a href="/">Home</a></li>
            <li class="active"> 
                <a href="/digitalmarketing/"></a>
            </li>
</>
```
There is no link-text in the subsequent breadcrumb items. I have specified 'breadcrumb' in the front matter of each page too.

I've spent ages trying to get this to work, I'd really appreciate any help with it? Thanks.
